README.md

```diff
@@ -3,26 +3,35 @@
 ## Install
 
 ### Prerequisites
-- cmake 3.10
+- cmake 3.10 or higher
 - GNU GCC with C++17 (gcc-8 or higher), LLVM Clang (clang-7 or higher), or AppleClang (clang-1100.0.33.8 or higher)
 - HTSlib
-- folly (optional)
-- Python 3 (for running integration tests)
 
-All can be installed with [brew](https://brew.sh) or [linuxbrew](https://linuxbrew.sh) (does not require root)
```

Comment:
I think it's better to keep this line (maybe just rephrase this a bit), it shows how to install the dependencies without root.